### DISCUSSION TITLE
How to bring back the one-test-per-line format in "short test summary info"?

### DISCUSSION THREAD

**Discussion by mdmintz** [pytest-dev/pytest#10438] [Q&A]
In the new ``pytest`` ``7.2.0`` "short test summary info", tests can take up one or two lines, like this:

<img width="1060" alt="Screen Shot 2022-10-26 at 9 15 39 AM" src="https://user-images.githubusercontent.com/6788579/198037073-49498c6e-fe45-4c5e-b5c6-72a0d63670b1.png">

But in the previous version, ``7.1.3``, there was only one line per test, which can be more readable:

<img width="1058" alt="Screen Shot 2022-10-26 at 9 17 51 AM" src="https://user-images.githubusercontent.com/6788579/198037435-1983972d-9da7-4342-a7dd-e62f286cdf57.png">

Is there a way to make the results show only one line per test again? (For me it is cleaner and more readable, as the single-line format makes it easier to find the test that I'm looking for, especially when I have hundreds of tests to scan through.)

**Comment by The-Compiler** [comment]
Is this from CI output? If so, this is probably #9933. Locally, the output will still be on one line.

There doesn't seem to be a way to customize it, I'm afraid (other than unsetting the `CI` variable, but I would advise against that, since it probably has some other effects as well).

**Reply by mdmintz** [reply]
This test suite was run locally on my Mac from a terminal.

**Reply by mdmintz** [reply]
Would that mean that pytest thinks my local Mac terminal is a CI server?

**Reply by The-Compiler** [reply]
Either that, or there is something else going wrong. I don't think this is an intended change. I'm assuming that `env | grep CI` in your terminal doesn't show anything?

**Reply by mdmintz** [reply]
Running ``env | grep CI`` does not show anything in my Mac terminal.

**Comment by jeffwright13** [comment]
> (For me it is cleaner and more readable, as the single-line format makes it easier to find the test that I'm looking for, especially when I have hundreds of tests to scan through.)

Not really a direct answer to your question, but you could try using something like pytest-html or pytest-tui to more easily sift through your hundreds of results.

**Reply by mdmintz** [reply]
I've gotten used to the new format. Even though each result could take up 1 or 2 lines, my brain can parse it out now.

And when dealing with hundreds of results, I just use the SeleniumBase dashboard/html combo:

[<img src="https://camo.githubusercontent.com/cedb5543e595fb8b6cc35b97b0427cf39e844490a27bc64f9a904c1322feaacd/68747470733a2f2f73656c656e69756d626173652e6769746875622e696f2f63646e2f696d672f64617368626f6172645f322e706e67" title="SeleniumBase" width="400">](https://github.com/seleniumbase/SeleniumBase/)
